src/www/ui/api/index.php

```diff
@@ -146,6 +146,7 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->put('/{id:\\d+}/permissions', UploadController::class . ':setUploadPermissions');
     $app->get('/{id:\\d+}/perm-groups', UploadController::class . ':getGroupsWithPermissions');
     $app->get('/{id:\\d+}/summary', UploadController::class . ':getUploadSummary');
+    $app->delete('/{id:\\d+}/licenses/{shortName:\\w+}/main', UploadController::class . ':removeMainLicense');
```

Comment:
Shortname can have a `-` and ` ` space in name.
```suggestion
    $app->delete('/{id:\\d+}/licenses/{shortName:[\\w\\- ]+}/main', UploadController::class . ':removeMainLicense');
```